docs/src/user.adoc

```diff
@@ -402,11 +402,14 @@ The `metrics` configuration defines how Cassandra Sidecar exposes metrics over J
 
 This section defines input validations for Cassandra keyspace and directory names which are used for SSTable imports. The following properties are defined:
 
+* `validator`: The implementation to use for the validation of Casandra inputs.
+** `class_name`: The name of the class implementing the CassandraInputValidator interface. Out of the box Cassandra Sidecar provides org.apache.cassandra.sidecar.utils.{RegexBasedCassandraInputValidator, FastCassandraInputValidator}.
+** `parameters`: Configuration parameters that are only applicable to the FastCassandraInputValidator implementation.
```

Comment:
Can you also add the new `FastCassandraInputValidator` based parameters `valid_terminations`, `valid_restricted_terminations` to docs?